src/components/collaborator-socio-card.tsx

```diff
@@ -0,0 +1,95 @@
+interface Cardprops {
+  type: "colaborador" | "socio";
+}
+
+const getGradientColor = (type: "colaborador" | "socio") => {
+  switch (type) {
+    case "colaborador":
+      return "#5069ED"; 
+    case "socio":
+      return "#ED7950"; 
+    default:
+      return "#000000"; 
+  }
+};
+
+const info = {
+  colaborador: {
+    name: ["colaborador"],
+    text: [
+      "Podes juntar-te a estes departamentos e contribuir para fazer o teu núcleo funcionar.",
+      "Vem fazer parte desta equipa e ajuda a tornar o CeSIUM melhor todos os dias.",
+    ],
+  },
+  socio: {
+    name: ["socio"],
+    text: [
+      "Participa em todos os nossos eventos gratuitamente, e usufrui de muitos mais benefícios.",
+      "Podes juntar-te a estes departamentos e contribuir para fazer o teu núcleo funcionar.",
+    ],
+  },
+};
+
+const getRandomText = (type: "colaborador" | "socio") => {
+  if (type === "socio") {
+    const texts = info[type].text;
+    return texts[Math.floor(Math.random() * texts.length)];
+  } else {
+    const texts = info[type].text;
+    return texts[Math.floor(Math.random() * texts.length)];
+  }
+};
+
+const CollaboratorSocioCard = ({ type }: Cardprops) => {
```

Comment:
```suggestion
const CollaboratorSocioCard = ({ type }: Cardprops) => {
```

I don't really like the component name but can't think of something better tbh.

Reply:
What about TeamMemberCard ?